### ISSUE TITLE
Clearing fields after adding employee should be done.

### ISSUE THREAD

**Issue by abhijithanandan** [tornotron/echno_attendance#26]

**Comment by abhijithanandan** [comment]
Issue arised becuase the `TextEditingController` instances are not being cleared after the form submission.
To resove the issue the code should be modified as follows:
```
ElevatedButton(
  onPressed: () async {
    String phoneNumber = "+${selectedCountry.phoneCode}${_phoneController.text.trim()}";

    await HrClass().createUser(
        userId: _idController.text.trim(),
        name: _nameController.text.trim(),
        email: _emailController.text.trim(),
        phoneNumber: phoneNumber,
        userRole: _roleController.text.trim(),
        isActiveUser: isActive);

    // Clear the controllers after form submission
    _idController.clear();
    _nameController.clear();
    _emailController.clear();
    _phoneController.clear();
    _roleController.clear();
  },
  child: const Text('Add Employee'),
),
```

**Comment by abhijithanandan** [comment]
Commit cfce440b0849008a5e92f26da0bae3db5b873190 has the solution implemented. @abinkjohny, kindly verify the solution and close this issue.

**Comment by abinkjohny** [comment]
We have a similar issue in the `update_details.dart` file which I resolved in the following way @abhijithanandan please verify...
`SizedBox(
                        width: double.infinity,
                        child: ElevatedButton(
                          onPressed: () async {
                            await HrClass().updateUser(
                              userId: _employeeIdController.text,
                              name: _employeeData!['name'],
                              email: _employeeData!['email'],
                              phoneNumber: _employeeData!['phoneNumber'],
                              userRole: _employeeData!['userRole'],
                              isActiveUser: _employeeData!['isActiveUser'],
                            );
                            if (context.mounted) {
                              ScaffoldMessenger.of(context).showSnackBar(
                                SnackBar(
                                  backgroundColor: Colors.green.shade600,
                                  content: const Text(
                                      'Details updated successfully!'),
                                ),
                              );
                              setState(() {
                                _employeeData = null;
                                _employeeIdController.clear();
                              });
                            }
                          },
                          child: const Text('Update Details'),
                        ),
                      )`

**Comment by abinkjohny** [comment]
@abhijithanandan has resolved this issue in commit [cfce440](https://github.com/tornotron/echno_attendance/commit/cfce440b0849008a5e92f26da0bae3db5b873190)
